src/api/docs/content/external/highlight.min.js

```diff
@@ -1,2 +1,400 @@
+    },_={className:"function.dispatch",relevance:0,keywords:{
+    _hint:["abort","abs","acos","apply","as_const","asin","atan","atan2","calloc","ceil","cerr","cin","clog","cos","cosh","cout","declval","endl","exchange","exit","exp","fabs","floor","fmod","forward","fprintf","fputs","free","frexp","fscanf","future","invoke","isalnum","isalpha","iscntrl","isdigit","isgraph","islower","isprint","ispunct","isspace","isupper","isxdigit","labs","launder","ldexp","log","log10","make_pair","make_shared","make_shared_for_overwrite","make_tuple","make_unique","malloc","memchr","memcmp","memcpy","memset","modf","move","pow","printf","putchar","puts","realloc","scanf","sin","sinh","snprintf","sprintf","sqrt","sscanf","std","stderr","stdin","stdout","strcat","strchr","strcmp","strcpy","strcspn","strlen","strncat","strncmp","strncpy","strpbrk","strrchr","strspn","strstr","swap","tan","tanh","terminate","to_underlying","tolower","toupper","vfprintf","visit","vprintf","vsprintf"]
+    },
+    begin:t.concat(/\b/,/(?!decltype)/,/(?!if)/,/(?!for)/,/(?!switch)/,/(?!while)/,e.IDENT_RE,t.lookahead(/(<[^<>]+>|)\s*\(/))
+    },m=[_,l,s,a,e.C_BLOCK_COMMENT_MODE,o,c],f={variants:[{begin:/=/,end:/;/},{
+    begin:/\(/,end:/\)/},{beginKeywords:"new throw return else",end:/;/}],
+    keywords:p,contains:m.concat([{begin:/\(/,end:/\)/,keywords:p,
+    contains:m.concat(["self"]),relevance:0}]),relevance:0},g={className:"function",
+    begin:"("+i+"[\\*&\\s]+)+"+d,returnBegin:!0,end:/[{;=]/,excludeEnd:!0,
+    keywords:p,illegal:/[^\w\s\*&:<>.]/,contains:[{begin:n,keywords:p,relevance:0},{
+    begin:d,returnBegin:!0,contains:[u],relevance:0},{begin:/::/,relevance:0},{
+    begin:/:/,endsWithParent:!0,contains:[c,o]},{relevance:0,match:/,/},{
+    className:"params",begin:/\(/,end:/\)/,keywords:p,relevance:0,
+    contains:[a,e.C_BLOCK_COMMENT_MODE,c,o,s,{begin:/\(/,end:/\)/,keywords:p,
+    relevance:0,contains:["self",a,e.C_BLOCK_COMMENT_MODE,c,o,s]}]
+    },s,a,e.C_BLOCK_COMMENT_MODE,l]};return{name:"C++",
+    aliases:["cc","c++","h++","hpp","hh","hxx","cxx"],keywords:p,illegal:"</",
+    classNameAliases:{"function.dispatch":"built_in"},
+    contains:[].concat(f,g,_,m,[l,{
+    begin:"\\b(deque|list|queue|priority_queue|pair|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array|tuple|optional|variant|function)\\s*<(?!<)",
+    end:">",keywords:p,contains:["self",s]},{begin:e.IDENT_RE+"::",keywords:p},{
+    match:[/\b(?:enum(?:\s+(?:class|struct))?|class|struct|union)/,/\s+/,/\w+/],
+    className:{1:"keyword",3:"title.class"}}])}}})();hljs.registerLanguage("cpp",e)
+    })();
```

Comment:
Is this really a minified version? The filename suggests so...

Reply:
Yes it is. The full version is even larger
![2024-10-01_19-58](https://github.com/user-attachments/assets/725b1a55-caa3-4e5b-851b-2f272db6d474)

Reply:
Maybe I force pushed away an error...